x-pack/legacy/plugins/uptime/public/components/functional/snapshot_heading.tsx

```diff
@@ -13,27 +13,17 @@ interface Props {
   total: number;
```

Comment:
```suggestion
interface Props {
  total: number;
}
```

Let's delete the obsolete `down` prop.

As an extension of this, we can modify the implementation of this component, which lives in `snapshot.tsx` in the same directory. We can end up with something like this, which is simpler (we can also delete some more `lodash.get` code):

```ts
export const SnapshotComponent: React.FC<SnapshotComponentProps> = ({ count, height, loading }) => (
  <ChartWrapper loading={loading} height={height}>
    <SnapshotHeading total={count.total} />
    <EuiSpacer size="xs" />
    <DonutChart
      up={count.up}
      down={count.down}
      height={SNAPSHOT_CHART_HEIGHT}
      width={SNAPSHOT_CHART_WIDTH}
    />
  </ChartWrapper>
);
```
To illustrate why I want to do this, here's the original code:

```ts
export const SnapshotComponent: React.FC<SnapshotComponentProps> = ({ count, height, loading }) => (
  <ChartWrapper loading={loading} height={height}>
    <SnapshotHeading down={get<number>(count, 'down', 0)} total={get<number>(count, 'total', 0)} />
    <EuiSpacer size="xs" />
    <DonutChart
      up={get<number>(count, 'up', 0)}
      down={get<number>(count, 'down', 0)}
      height={SNAPSHOT_CHART_HEIGHT}
      width={SNAPSHOT_CHART_WIDTH}
    />
  </ChartWrapper>
);
```

### Other references

This would also necessitate deleting `down` props from implementations in tests in `snapshot_heading.test.tsx` and `chart_wrapper.test.tsx`.